### PR TITLE
Area of a room

### PR DESCRIPTION
**Pull request by ranjit-t**
here is the solution in index.js

### CODE REVIEW

File: projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+if(isNaN(area))
+{
+  alert(`make sure you enter right values`);
+    
+}
+else {
+    alert(`The area of the room is ${area}`);
+}
+
+
```
Comment (StefanoConsonni) [inline projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js]: Please make sure to use const variables whenever possible. You should also install prettier and set it up properly (turn on the "format on save option" in the vscode settings).

File: projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+let length = parseInt(prompt("Enter Length of the room in meters"));
+let width = parseInt(prompt(" Super, Now Enter Width of the room in meters"));
+
+let area = length*width;
+
+
+
+if(isNaN(area))
+{
+  alert(`make sure you enter right values`);
```
Comment (StefanoConsonni) [inline projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js]: In this case there is no need to use the backticks. Please convert it to double quotes.

File: projects/m1/002-bottle-deposits/solutions/ranjit-t/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+const amount1 = parseInt(prompt("Enter number of containers holding one liter or less"));
+const amount2 = parseInt(prompt("Enter number of containers holding more than one liter"));
+
+const refund = parseFloat((amount1*0.10)+(amount2*0.25)).toFixed(2);
+
+
+
+if(isNaN(refund))
+{
+  alert(`make sure you enter right values`);
```
Comment (StefanoConsonni) [inline projects/m1/002-bottle-deposits/solutions/ranjit-t/index.js]: In this case there is no need to use the backticks. Please convert it to double quotes.

File: projects/m1/002-bottle-deposits/solutions/ranjit-t/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    
+}
+else {
+    alert(`Here is your refund amount : ${refund}$`);
+}
```
Comment (StefanoConsonni) [inline projects/m1/002-bottle-deposits/solutions/ranjit-t/index.js]: Please turn on the vscode setting that allows the editor to automatically adds an empty line at the end of every file when you save

**Review by StefanoConsonni — CHANGES_REQUESTED** [review]
Good job overall. Please apply the requested changes

**Comment by ranjit-t** [comment]
Yes please, check now! is it okay ?

File: projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+
+if(isNaN(area))
+{
+  alert("make sure you enter right values");
+    
+}
+else {
+    alert(`The area of the room is ${area}`);
+}
```
Comment (StefanoConsonni) [inline projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js:15]: The code is not following the prettier  formatting rules. You should install prettier and set it up properly (turn on the "format on save option" in the vscode settings).

File: projects/m1/001-area-of-a-room/solutions/ranjith-thota/solution.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+
+if(isNaN(area))
+{
+  alert(`make sure you enter right values`);
+    
+}
+else {
+    alert(`The area of the room is ${area}`);
+}
```
Comment (StefanoConsonni) [inline projects/m1/001-area-of-a-room/solutions/ranjith-thota/solution.js:15]: The code is not following the prettier formatting rules.

File: projects/m1/002-bottle-deposits/solutions/ranjit-t/index.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+
+if(isNaN(refund))
+{
+  alert("make sure you enter right values");
+    
+}
+else {
+    alert(`Here is your refund amount : ${refund}$`);
+}
```
Comment (StefanoConsonni) [inline projects/m1/002-bottle-deposits/solutions/ranjit-t/index.js:16]: The code is not following the prettier formatting rules.

**Review by StefanoConsonni — CHANGES_REQUESTED** [review]

File: projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js (hunk 5 above)
Comment (StefanoConsonni) [reply projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js:15]: Also please submit your solutions in a folder named ranjithtota (in the solutions folder)

**Review by StefanoConsonni — CHANGES_REQUESTED** [review]

**Pull request closed without merging**